packages/broker/src/ConfigWizard.ts

```diff
@@ -82,200 +87,168 @@ const DefaultConfig: Config = {
                 }
             }
         },
-    }
+    },
 }
 
-export class ConfigWizard{
-    config: Config
-    // required to mock results on tests
-    inquirer: inquirer.Inquirer = inquirer
-
-    constructor() {
-        this.config = DefaultConfig
-    }
-
-    private async inquirerSinglePrompt(prompt: inquirer.Question | inquirer.ListQuestion | inquirer.CheckboxQuestion) {
-        const answers = await this.inquirer.prompt([prompt])
-        return answers[prompt.name!]
-    }
-
-    private async promptNumberWithDefault(prompt: inquirer.InputQuestion, defaultValue: number): Promise<number>{
-        const valueString = await this.inquirerSinglePrompt(prompt)
-        const value = valueString === '' ? defaultValue: parseInt(valueString)
-
-        if (Number.isNaN(value) || !Number.isInteger(value)) {
-            logger.warn(`Non-numeric value [${valueString}] provided`)
-            return await this.promptNumberWithDefault(prompt, defaultValue)
-        }
+const promptValues = {
```

Comment:
Would maybe give some more specific name? Or just two constants, e.g. `PRIVATE_KEY_SOURCE_GENERATE` and `PRIVATE_KEY_SOURCE_IMPORT`